bin/doctest

```diff
@@ -14,9 +14,15 @@ set -f ; shopt -u nullglob
 prefix="$(get comment-prefix)"
 opening="$(get opening-delimiter)"
 closing="$(get closing-delimiter)"
+module="$(get module-type)"
+
+if [[ "$module" == esm ]] && [ "$(node_major_version)" -lt 9 ] ; then
```

Comment:
```suggestion
if [[ "$module" == esm ]] && [[ "$(node_major_version)" -lt 9 ]] ; then
```

Reply:
Oh! I thought `[[` doesn't support `-lt` for some reason.

Reply:
My understand is that `[[` supports a superset of the `[` functionality. The main difference between the two is that `[[` is a language feature whereas `[` is a program:

```console
$ which -a [
/bin/[

$ which -a [[
```

By virtue of being a language feature `[[` gets to do things that are impossible for programs. As a result, `[[` has fewer traps than `[`.